src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -16,32 +16,211 @@ redirects:
+
+        The monitor uses Google Chrome browser. You can also use a variety of [third-party modules](/docs/synthetics/new-relic-synthetics/scripting-monitors/scripted-monitor-runtime-environment#runtime-table) to build your custom monitor.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        API tests
+
+        API name: `SCRIPT_API`
+      </td>
+
+      <td>
+        API tests are used to monitor your API endpoints. This can ensure that your app server works in addition to your website. New Relic uses the [`http-request` module](https://github.com/request/request) internally to make HTTP calls to your endpoint and validate the results.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+You can [add and edit synthetic monitors](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors) directly in our UI. You can also create some synthetic monitors with [Terraform](https://developer.newrelic.com/automate-workflows/get-started-terraform/). To learn how, watch this short YouTube video (approx. 3:25 minutes).
+
+<Video
+  id="WIVOgYlID-E"
+  type="youtube"
+/>
+
+## Advanced testing with scripted browsers [#advanced]
 
-Synthetic monitoring allows you to proactively monitor your website or API endpoint to ensure your content is not only available, but fully functional. Synthetic monitoring browser tests send real, Selenium-powered Google Chrome browsers to your site from locations around the world to ensure your content is always up, everywhere.
+Synthetic monitoring allows you to proactively monitor your website or API endpoint to ensure your content is not only available, but fully functional. [Scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts) send real, Selenium-powered Google Chrome instances to your site from locations around the world, so you're confident that your content is always up, everywhere.
 
-[Scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts) expand your testing capabilities, so you can test uncommon user flows or beta-test complex procedures. For example, ensure your users are able to sign up for your newsletter, add an item to their cart, or search for and find a piece of critical content with a simple JavaScript-like language. Test your backend with API monitors, which allow you to run scripted tests against any API endpoint.
+Scripted browsers expand your testing capabilities, so you can test uncommon user flows or beta-test complex procedures. For example, ensure your users are able to sign up for your newsletter, add an item to their cart, or search for and find a piece of critical content with a simple JavaScript-like language. Test your backend with API monitors, which allow you to run scripted tests against any API endpoint.
```

Comment:
In the second sentence, would you consider adding "you can" before "ensure"? I only bring it up because when I read through it I'm expecting a procedure to follow the "ensure" part (ie: ensure your users are able to sign up... by"), but then it kind of turns into a list of things you can do instead. Again, just a small thing that's not gating.